src/docs/product/accounts/quotas/manage-replay-quota.mdx

```diff
@@ -0,0 +1,86 @@
+
+## Before You Begin: Check Your Quota Usage {#3-replay-usage-stats}
+
+You can look at your replays in aggregate in the "Usage Stats" tab of the **Stats** page to understand the breakdown of your incoming replays and see which projects are consuming your [quota](The monthly number of events that you pay Sentry to track.). This may help you figure out where you need to further fine-tune your SDK filters and configuration.
+
+This page is accessible to all members of your organization, so Owners in your Sentry org will be able to share it with the developers directly responsible for a given project. You’ll also be able to use this page to assess whether the changes you've made are having the desired effect.
+
+### How can I See a Breakdown of Incoming Replays?
+
+The Usage Stats tab displays details about the total amount of data Sentry has received across your entire organization for up to 90 days. The page breaks down the events (by project) into three categories: accepted, dropped, or filtered. Only accepted replays affect your [quota](The monthly number of events that you pay Sentry to track.):
+
+![Usage Stats page showing replays](usage-stats-replay.png)
+
+### Which Projects Are Consuming My Quota?
+
+The "Project" table in the "Usage Stats" tab of the Stats page breaks down your data by project, so you can identify the ones that are consuming the most [quota]. Clicking on the settings icon next to a project name in the table, will open the project's settings page, where you can manage its [inbound filters](#2-inbound-data-filters)and rate limits.
+
+##  Adjusting Quotas
+
+<Note>
+
+Quotas can only be updated by a Billing or Owner-level member of your Sentry organization.
+
+</Note>
+
+Once your replay volume is approaching or has exceeded its [quota](The monthly number of events that you pay Sentry to track.), teammates with the "Owner" organization permission level will start receiving [notification](/product/alerts/notifications/#quota-notifications) emails. They'll then be able to choose to [increase](#increasing-quotas) or [decrease](#decreasing-quotas) your quota.
+
+If this is your first time exceeding quota and you're on a paid plan, however, you'll be entered into a one-time grace period. Learn more about the grace period in this [Help article](https://help.sentry.io/account/billing/what-happens-when-i-run-out-of-event-capacity-and-a-grace-period-is-triggered/).
+
+### Increasing Quotas
+
+If you're dropping replays that you want to keep because you've exceeded your [quota](The monthly number of events that you pay Sentry to track.), you can add to your quota at any time during your billing period by either increasing your[reserved](#reserved-volume) or [on-demand](#on-demand-budget) quota.
+
```

Comment:
```suggestion
If you're dropping replays that you want to keep because you've exceeded your quota, you can add to your quota at any time during your billing period by either increasing your [reserved](#reserved-volume) or [on-demand](#on-demand-budget) quota.

```